regifter.sql

```diff
@@ -15,104 +15,114 @@ CREATE DATABASE regifter;
 -- giver - string
 -- value - integer
 -- previously_regifted boolean
-
+CREATE TABLE gifts (id SERIAL PRIMARY KEY, gift TEXT NOT NULL, giver TEXT, value INT, previously_regifted BOOLEAN);
```

Comment:
Get in the habit of knocking these long commands onto multiple lines.  Its convention in SQL and makes code easier to read.